bigquery_etl/query_scheduling/templates/airflow_dag.j2

```diff
@@ -304,6 +316,69 @@ with DAG('{{ name }}', default_args=default_args{%+ if schedule_interval != None
     {% endif -%}
     {% endfor -%}
 
+    {% for table_sensor_task in task.depends_on_tables_existing -%}
+    {% set project_id, dataset_name, table_name = table_sensor_task.table_id.split('.') -%}
+    {{ table_sensor_task.task_id }} = BigQueryTableExistenceSensor(
+        task_id={{ table_sensor_task.task_id | format_repr }},
+        project_id={{ project_id | format_repr }},
+        dataset_id={{ dataset_name | format_repr }},
+        table_id={{ table_name | format_repr }},
+        gcp_conn_id='google_cloud_shared_prod',
+        deferrable=True,
```

Comment:
I opted to use the newer [deferrable operator](https://airflow.apache.org/docs/apache-airflow/2.7.3/authoring-and-scheduling/deferring.html) approach (rather than `mode="reschedule"`) since the BigQuery table sensors support it.  This might be a bit of a risk, since I think this could be the very first use of deferrable operators in our Airflow setup, but it seemed worth trying.